api/nakadi-event-bus-api.yaml

```diff
@@ -798,6 +798,141 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  /subscriptions/{subscription_id}/events:
+    get:
+      tags:
+        - stream-api
+        - managed-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: |
+        ---
+      parameters:
+        parameters:
+          - name: subscription_id
+            in: path
+            type: string
+            description: Id of subscription
+            required: true
+          - name: commit_timeout
+            in: query
+            description: |
+              The amount of time allowed between commits. If no commit is performed within this amount of time - the
+              connection will be closed from Nakadi side. Specified in seconds.
+            type: integer
+            format: int32
+            required: false
+            default: 60
+          - name: max_uncommitted_event_count
+            in: query
+            description: |
+              The amount of uncommitted events Nakadi will stream before pausing the stream. When in paused state and
+              commit comes - the stream will continue.
+              Specifying 0 for this property will make the max uncommitted event count unlimited (the stream will never
+              be paused).
+              The property is ignored when autocommit is true for the subscription.
+            type: integer
+            format: int32
+            required: false
+            default: 10000
+          - name: batch_limit
+            in: query
+            description: |
+              Maximum number of `Event`s in each chunk (and therefore per partition) of the stream.
+
+              * If 0 or unspecified will buffer Events indefinitely and flush on reaching of
+              `batch_flush_timeout`.
+            type: integer
+            format: int32
+            required: false
+            default: 1
+          - name: stream_limit
+            in: query
+            description: |
+              Maximum number of `Event`s in this stream (over all partitions being streamed in this
+              connection).
+
+              * If 0 or undefined, will stream batches indefinitely.
+
+              * Stream initialization will fail if `stream_limit` is lower than `batch_limit`.
+            type: integer
+            format: int32
+            required: false
+            default: 0
+          - name: batch_flush_timeout
+            in: query
+            description: |
+              Maximum time in seconds to wait for the flushing of each chunk (per partition).
```

Comment:
Do you think we should make explicit some constraints? For example, it should be lower than `commit_timeout`. Otherwise we could create a stream that is bricked, as users won't receive events before `batch_flush_timeout` so they won't have a proper `cursor` to commit (remembering that clients only commit cursors as an opaque structure).